### PR TITLE
🌱 Fix CAPI release to v1.3.0-beta.1 for main branch test and uplift go to v1.19

### PR DESCRIPTION
**Pull request by kashifest**

### CODE REVIEW

File: lib/releases.sh [hunk 1]
```diff
@@ -43,6 +43,11 @@ else
   export CAPIRELEASE="${CAPIRELEASE:-$(get_latest_release "${CAPIRELEASEPATH}" "v1.2.")}"
 fi
 
+# Remove the following line when CAPI release v1.3.0 is available
+if [ "${CAPM3RELEASEBRANCH}" == "main" ]; then
+ export CAPIRELEASE="v1.3.0-beta.0"
```
Comment (furkatgofurov7) [inline lib/releases.sh:48]:
```suggestion
 export CAPIRELEASE="v1.3.0-beta.1"
```

**Review by furkatgofurov7 — CHANGES_REQUESTED** [review]

File: lib/releases.sh (hunk 1 above)
Comment (mboukhalfa) [inline lib/releases.sh:48]: Are not we using beta.1 in capm3 ?

**Review by furkatgofurov7 — APPROVED** [review]
/retitle 🌱 Fix CAPI release to v1.3.0-beta.1 for main branch test and uplift go to v1.19
/approve

**Comment by Rozzii** [comment]
/hold
feel free to remove the hold when you have sorted out the beta.0/beta.1 situation.

File: lib/releases.sh (hunk 1 above)
Comment (kashifest) [reply lib/releases.sh]: Done

**Comment by kashifest** [comment]
/hold cancel
@furkatgofurov7 we need to force merge it right ?

**Comment by kashifest** [comment]
@Rozzii PTAL again

**Comment by furkatgofurov7** [comment]
> @furkatgofurov7 we need to force merge it right ?

this or other two (capm3/ipam). I would prefer this to be merged.

**Comment by furkatgofurov7** [comment]
Wait, would this pass 1-2 tests with go version 1.19? We might need a condition check around it

**Comment by furkatgofurov7** [comment]
To be sure, release branches CI tests won't be broken, otherwise fine with merging it manually

/test-ubuntu-integration-release-1-2
/test-centos-integration-release-1-1

**Comment by furkatgofurov7** [comment]
release-1-2 has passed, I believe release-1-1 should also be fine, but we were not able to verify it due to CI being too shaky (vm creation problem due to internal infra issues in Cleura) ):

**Comment by furkatgofurov7** [comment]
Force merging this to bypass the CI (deadlock) in https://github.com/metal3-io/ip-address-manager/pull/164 and https://github.com/metal3-io/cluster-api-provider-metal3/pull/774 since the latter already tested the changes in this PR.

**Pull request merged**